### PR TITLE
UX improvements in game text in attic and basement, fixed bug with ea…

### PR DESCRIPTION
**Pull request by mccarrom**
…gleWatching after looking at the eagle without the umbrella.

### CODE REVIEW

File: parser.go [hunk 1]
```diff
@@ -104,6 +104,7 @@ func moveToRoom(exit string) {
 	if eagleWatching {
 		fmt.Printf("AGH! The eagle is taking his revenge!\n")
 		tauntTheEagle()
+		eagleWatching = false
```
Comment (ewk) [inline parser.go]: Do we need this? The first thing tauntTheEagle() does is set eagleWatching = false.  

The only place eagleWatching is set true is lookAtEagle, which is only called if the player is in the Yard. 
From there you can useTheUmbrella or you can tauntEagle. If you try to keep going and ignore the eagle, tauntEagle is called for you.

File: parser.go [hunk 2]
```diff
@@ -146,6 +146,9 @@ func moveToRoom(exit string) {
 				if curRoom.Name == "Downstairs Hallway" && val.Name == "Staircase" {
 					climbTheStairs()
 				}
+				if curRoom.Name == "Basement Lab" {
+					fmt.Println("There's a message above the door here:\nThe unshrink function only works with the mirror in the attic.\n")
```
Comment (ewk) [inline parser.go]: We're going to hit the user with this every time? Shouldn't it be slightly hidden as a room feature?

File: parser.go [hunk 3]
```diff
@@ -146,9 +146,7 @@ func moveToRoom(exit string) {
 				if curRoom.Name == "Downstairs Hallway" && val.Name == "Staircase" {
 					climbTheStairs()
 				}
-				if curRoom.Name == "Basement Lab" {
-					fmt.Println("There's a message above the door here:\nThe unshrink function only works with the mirror in the attic.\n")
```
Comment (ewk) [inline parser.go]: Oh I see; it got dropped on the very next commit.

**Review by BethMyre — APPROVED** [review]
looks good to me

**Pull request merged**